src/main/java/controller/InputView.java

```diff
@@ -0,0 +1,63 @@
+package controller;
+
+import java.util.Arrays;
+import java.util.List;
+import java.util.Scanner;
+
+public class InputView {
+    
+    public static final char MATCH_LOTTO_DELIMITER = ',';
+    
+    public final Scanner SC;
```

Comment:
```suggestion
    private static final char MATCH_LOTTO_DELIMITER = ',';
    
    private final Scanner SC;
```
InputView 내부에서만 사용되기 때문에 private 으로 선언해도 좋을 것 같아요~

Reply:
[refactor: 외부에서 사용하지 않는 멤버를 private으로 캡슐화](https://github.com/woowacourse/java-lotto/pull/527/commits/e8fd3816dbfb2ae6b2b0eeeb6cd9ad88f0b45824)

코드 자동 생성 기능 때문에 이렇게 작성된 것 같아요! 감사합니다.